### ISSUE TITLE
<List> functionality is missing example ListRemoveByValue()

### ISSUE THREAD

**Issue by mohamedragabdahab** [aerospike-community/aerospike-client-php#33]
Missing Most of List functionality such as ListRemoveByValue() etc and constants such as 
ADD_DATA_UNIQUE

**Comment by aerospikerobertmarks** [comment]
Thanks for your interest, the feature request has been logged.

I'm not sure I understand what  you mean with the `ADD_DATA_UNIQUE` flag, Are you referring to the flags mentioned at https://www.aerospike.com/docs/guide/cdt-list-ops.html#modify-flags ?

**Comment by mohamedragabdahab** [comment]
yes exactly, i would love if you guys can support all the <LIST> functionality

**Comment by Thomas-Heniart** [comment]
Hi!

Flags like `ADD_DATA_UNIQUE` are still missing for operations on lists.
Could you provide any documentation or example on how to proceed?

Kind regards,
